tests/detect-flowbits/test.rules

```diff
@@ -0,0 +1,13 @@
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,fb1; sid:1;)
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,fb2; sid:2;)
+alert ip any any -> any any (msg:"Flowbit isset ored flowbits"; flowbits:isset,fb3|fb4; sid:3;)
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,fb5; sid:5;)
+alert ip any any -> any any (msg:"Flowbit isset ored flowbits"; flowbits:isset,fb5|fb6; sid:6;)
+alert ip any any -> any any (msg:"Flowbit isnotset ored flowbits"; flowbits:isnotset, fb1 | fb2 ; sid:7;)
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,myflow; sid:8;)
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:set,myflow2; sid:9;)
+alert ip any any -> any any (msg:"Flowbit unset"; flowbits:unset,myflow2; sid:10;)
+alert ip any any -> any any (msg:"Flowbit set"; flowbits:noalert; flowbits:toggle,myflow2; content:"GET "; sid:11;)
```

Comment:
After a `toggle` rule, we should probably have to rule to check if the flowbit was indeed toggled.